playbooks/robusta_playbooks/oom_killer.py

```diff
@@ -116,23 +143,31 @@ def pod_oom_killer_enricher(event: PodEvent, params: OomKillParams):
         logging.error(f"could not find OOMKilled status in pod {pod.metadata.name}")
         container_name = None
     else:
+        container_labels: List[Tuple[str, str]] = []
+        container_labels.extend(labels)
         container_name = oomkilled_container.container.name
         requests, limits = PodContainer.get_memory_resources(oomkilled_container.container)
-        labels.append(("Container name", container_name))
+        container_labels.append(("Container name", container_name))
         memory_limit = "No limit" if not limits else f"{limits}MB limit"
         memory_requests = "No request" if not requests else f"{requests}MB request"
-        labels.append(("Container memory", f"{memory_requests}, {memory_limit}"))
+        container_labels.append(("Container memory", f"{memory_requests}, {memory_limit}"))
         oom_killed_status = oomkilled_container.state
         if oom_killed_status.terminated.startedAt:
-            labels.append(("Container started at", oom_killed_status.terminated.startedAt))
+            container_labels.append(("Container started at", oom_killed_status.terminated.startedAt))
         if oom_killed_status.terminated.finishedAt:
-            labels.append(("Container finished at", oom_killed_status.terminated.finishedAt))
-    table_block = TableBlock(
-        [[k, v] for (k, v) in labels],
-        ["field", "value"],
-        table_name="*Pod and Node OOMKilled data*",
-    )
-    finding.add_enrichment([table_block])
+            container_labels.append(("Container finished at", oom_killed_status.terminated.finishedAt))
+
+        if params.container_memory_graph:
+            table_block = TableBlock(
```

Comment:
Only the graph should be controlled by the `container_memory_graph` parameter
 The table should always be added

Reply:
done